packages/ripple-binary-codec/src/enums/index.ts

```diff
@@ -70,22 +71,34 @@ export class Bytes {
 class BytesLookup {
   constructor(types: Record<string, number>, readonly ordinalWidth: number) {
     Object.entries(types).forEach(([k, v]) => {
-      this[k] = new Bytes(k, v, ordinalWidth)
-      this[v.toString()] = this[k]
+      this.add(k, v)
     })
   }
 
+  /**
+   * Add a new name value pair to the BytesLookup. Does not work properly if either name or value already
+   * exist within the BytesLookup.
```

Comment:
```suggestion
   * Add a new name value pair to the BytesLookup. Does not work properly if either name or value already
   * exist within the BytesLookup.
```
If so, there should be additional checks for if `name` or `value` already exist, to throw a nicer error.